bin/MadGraph5_aMCatNLO/cards/production/13TeV/MuMuJet_mll_0to60_LO/dyellell01234j_5f_LO_MLM_run_card.dat

```diff
@@ -0,0 +1,270 @@
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.1880  = scale            ! fixed ren scale
+ 91.1880  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.1880  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 0        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
+ 1        = highestmult      ! for ickkw=2, highest mult group
+ 1        = ktscheme         ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+ 1        = alpsfact         ! scale factor for QCD emission vx
+ F        = chcluster        ! cluster only according to channel diag
+ F        = pdfwgt           ! for ickkw=1, perform pdf reweighting
+ 5        = asrwgtflavor     ! highest quark flavor for a_s reweight
+ T        = clusinfo         ! include clustering tag in output
+ 3.0      = lhe_version       ! Change the way clustering information pass to shower.        
+#*********************************************************************
+#**********************************************************
+#
+#**********************************************************
+# Automatic ptj and mjj cuts if xqcut > 0
+# (turn off for VBF and single top processes)
+#**********************************************************
+   F  = auto_ptj_mjj  ! Automatic setting of ptj and mjj
+#**********************************************************
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15  = bwcutoff      ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#**********************************************************
+   F  = cut_decays    ! Cut decay products 
+#*************************************************************
+# Number of helicities to sum per event (0 = all helicities)
+# 0 gives more stable result, but longer run time (needed for
+# long decay chains e.g.).
+# Use >=2 if most helicities contribute, e.g. pure QCD.
+#*************************************************************
+   0  = nhel          ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+ 20  = ptj       ! minimum pt for the jets 
```

Comment:
Your analysis level minimum jet pt cuts is bigger than 20 GeV?

Reply:
Yes, we will have pT cut > 20GeV

Reply:
Then I'd suggest you loosen the ptj cut to 10 or 15, 20 GeV ME level cut is too tight

Reply:
Could you explain why 20GeV is too tight?

Reply:
Could you explain why a 20GeV cut is too tight? Thanks

Reply:
There could be some smearing effects.
ME level ptj = 19 -> RECO level ptj = 21
Things like this could happen especially for the jets where the momentum resolution is not so great compared to the leptons.

Reply:
Thanks. Then I think I should have make it clearer in my response earlier. Sorry for the confusion.
What I meant a pT cut > 20GeV was, we would apply a cut like 30GeV above, and this cut value is larger than 20GeV. 
Since we would like to have a higher cut than what we put in the card (or at ME level), I think it's safe to use 20GeV.